ml-prims/src/hmm/magma/magma_test_utils.h

```diff
@@ -0,0 +1,372 @@
+#pragma once
+
```

Comment:
General comment for this file:
I think this file belongs inside `ml-prims/test` folder, as per its name?

Reply:
I should have rather named it magma_utils, which is done now ^^